remote/src/test/scala/org/apache/pekko/remote/classic/RemotingSpec.scala

```diff
@@ -77,8 +77,10 @@ object RemotingSpec {
 
   val cfg: Config = ConfigFactory.parseString(s"""
     common-ssl-settings {
-      key-store = "${getClass.getClassLoader.getResource("keystore").getPath}"
-      trust-store = "${getClass.getClassLoader.getResource("truststore").getPath}"
+      key-store = "${Paths.get(getClass.getClassLoader.getResource("keystore").toURI).normalize().toString
```

Comment:
Would it be possible to create a private function that takes a `resourceName: String` and that returns the normalizedPath? Slightly tidier to reuse the normalize and replace here.